kubernetes_deploy/production/deployment.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: laa-fala
+spec:
+  replicas: 2
```

Comment:
Should the number of production replicas be 3, as per cla_public?
I think we can reasonably downscale memory requirements for our components.

Reply:
Good shout